test/grpc/error_test.exs

```diff
@@ -0,0 +1,33 @@
+defmodule GRPC.RPCErrorTest do
+	use ExUnit.Case
+
+  alias GRPC.RPCError
+
+  describe "is_rpc_error/2 guard clause" do
+    import GRPC.RPCError
+
+    test "returns true for GRPC.RPCError structs with same status" do
+      exception = GRPC.RPCError.new(:unknown)
+
+      assert is_rpc_error(exception, 2)
+    end
+
+    test "returns false for GRPC.RPCError structs with different status" do
+      exception = GRPC.RPCError.new(:unknown)
+
+      refute is_rpc_error(exception, 1)
+    end
+
+    test "returns false for others types of input" do
+      exception = %{status: :unknown}
+
+      refute is_rpc_error(exception, 2)
+    end
+  end
+
+  test "new/1 create GRPC.RPCError struct with a default message" do
```

Comment:
```suggestion
  test "new/1 creates GRPC.RPCError struct with a default message" do
```

By the way, I'd have written this as:

```elixir
describe "new/1" do
  test "creates ... " do
    ...
  end
end
```

Reply:
Fixed [f48e9d6](https://github.com/elixir-grpc/grpc/pull/314/commits/f48e9d6677db4caa481119b67ec5ec4aecd5d917)